ai-server/main.py

```
@@ -299,7 +287,67 @@ async def query_document(request: QueryRequest):
+                {"done": True, "answer": "관련 내용을 문서에서 찾을 수 없습니다.", "sources": []},
+                ensure_ascii=False
+            )
+            yield f"data: {payload}\n\n"
+
+        return StreamingResponse(
+            empty_stream(),
+            media_type="text/event-stream",
+            headers={"Cache-Control": "no-cache", "X-Accel-Buffering": "no"}
+        )
+
+    async def token_stream():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `token_stream`

(ANN202)